Composer/packages/server/src/models/connector/csharpBotConnector.ts

```diff
@@ -109,3 +222,16 @@ export class CSharpBotConnector implements IBotConnector {
     });
   };
 }
+
+process.on('SIGINT', () => {
+  CSharpBotConnector.stopAll('SIGINT');
+  process.exit(0);
```

Comment:
Still don't need to do this. The process is already exiting.

Reply:
I try to remove it, and find the Debugger won't exit automatically. port 9228 is still listened. The next 'yarn start:dev' will be blocked.
![image](https://user-images.githubusercontent.com/21174180/70313164-0b0be280-1850-11ea-9f17-c70d23d93ca9.png)

Reply:
Weird!

Well let's just clean this up a bit:

```ts
const cleanup = (signal) => {
  CSharpBotConnector.stopAll(signal);
  process.exit(0);
}
['SIGINT', 'SIGTERM', ...].forEach((signal) => {
  process.on(signal, cleanup.bind(null, signal))
});
```